juju/juju
# "Added_synonym_application_timeout_unit_for_run"

## Please provide the following details to expedite Pull Request review:

----

## Description of change
Why is this change needed?
Options for run commands like application,timeout,unit can be abbreviated for ease of use in command line

## QA steps
How do we verify that the change works?
Configured LXD, created controller and deployed simple application mediawiki.
Verified after making changes to the code using juju CLI - run command with combinations.
for eg: juju run --app=mysql -u mysql/0  "ls"
## Documentation changes
Does it affect current user workflow? CLI? API?
N/A
## Bug reference
Does this change fix a bug? Please add a link to it
https://bugs.launchpad.net/juju/+bug/1751321.
